tests/unit/test_helpers.py

```diff
@@ -499,6 +500,16 @@ def test_ab_init_active(self):
             ab_init(response)
             response.set_cookie.assert_not_called()
 
+    def test_get_extra_labels(self):
+        """Test extra_label extraction from form object"""
```

Comment:
nit: missing the period at the end of the line.

```suggestion
        """Test extra_label extraction from form object."""
```